src/Factory/AdapterFactory.php

```diff
@@ -224,4 +231,10 @@ private function guardFactoryClassImplementation($adapterFactory, $label)
             self::SUPPORT_ISSUE_TRACKER => $issueTracker,
         ];
     }
+
+    private  function getBaseAdapter($name) {
+      $array = explode(':', $name);
+      return $array[0];
+    }
+
```

Comment:
Only the GIT url will not work as the API and web URL also need to be configured (see mentioned issue).